.gitmodules

```
@@ -1,11 +1,11 @@
 [submodule "tests"]
 	path = tests/testdata
-	url = https://github.com/etclabscore/tests
+	url = https://github.com/ethereum/tests
 	shallow = true
 [submodule "evm-benchmarks"]
 	path = tests/evm-benchmarks
 	url = https://github.com/ipsilon/evm-benchmarks
 	shallow = true
 [submodule "tests-etc"]
 	path = tests/testdata-etc
-	url = https://github.com/etclabscore/tests-etc
+	url = https://github.com/etclabscore/tests
```

This should have `shallow = true`. The repo is big and the story is long.

---

```suggestion
	url = https://github.com/etclabscore/tests
	shallow = true
```

@meowsbits it's your commit. Accept it

---

Resolved by [460091f](https://github.com/etclabscore/core-geth/pull/510/commits/460091f7268e2606cab870c43103ac410cda26ec)